<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NguyenTrung" failures="4" tests="12" name="Test" time="75.457" errors="0" timestamp="2022-04-12T12:39:31 ICT">
  <testcase classname="TestASM2.Test.TestLoginExcelData" name="multiLogin" time="3.215"/>
  <testcase classname="TestASM2.Test.TestLoginExcelData" name="multiLogin" time="3.538"/>
  <testcase classname="TestASM2.Test.TestLoginExcelData" name="multiLogin" time="0.784"/>
  <testcase classname="TestASM2.Test.TestLoginExcelData" name="multiLogin" time="0.664"/>
  <testcase classname="TestASM2.Test.TestLoginExcelData" name="multiLogin" time="3.475">
    <failure type="java.lang.AssertionError" message="expected [Online Entertainment] but found [Login Online Entertainment]">
      <![CDATA[java.lang.AssertionError: expected [Online Entertainment] but found [Login Online Entertainment]
at TestASM2.Test.TestLoginExcelData.multiLogin(TestLoginExcelData.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- multiLogin -->
  <testcase classname="TestASM2.Test.TestLoginExcelData" name="multiLogin" time="3.186">
    <failure type="java.lang.AssertionError" message="expected [Online Entertainment] but found [Login Online Entertainment]">
      <![CDATA[java.lang.AssertionError: expected [Online Entertainment] but found [Login Online Entertainment]
at TestASM2.Test.TestLoginExcelData.multiLogin(TestLoginExcelData.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- multiLogin -->
  <testcase classname="TestASM2.Test.TestInsertUser" name="multiInsert" time="5.721"/>
  <testcase classname="TestASM2.Test.TestInsertUser" name="multiInsert" time="5.599"/>
  <testcase classname="TestASM2.Test.TestUpdateUser" name="multiInsert" time="5.016"/>
  <testcase classname="TestASM2.Test.TestUpdateUser" name="multiInsert" time="5.072">
    <failure type="java.lang.AssertionError" message="expected [User Updated!!] but found [User Deleted!!]">
      <![CDATA[java.lang.AssertionError: expected [User Updated!!] but found [User Deleted!!]
at TestASM2.Test.TestUpdateUser.multiInsert(TestUpdateUser.java:80)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- multiInsert -->
  <testcase classname="TestASM2.Test.TestDeleteUser" name="multiInsert" time="5.073"/>
  <testcase classname="TestASM2.Test.TestDeleteUser" name="multiInsert" time="4.951">
    <failure type="java.lang.AssertionError" message="expected [Error: Error while committing the transaction] but found [User Deleted]">
      <![CDATA[java.lang.AssertionError: expected [Error: Error while committing the transaction] but found [User Deleted]
at TestASM2.Test.TestDeleteUser.multiInsert(TestDeleteUser.java:75)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- multiInsert -->
</testsuite> <!-- Test -->
